server/database/queries/index.js

```diff
@@ -0,0 +1,7 @@
+const {checkEmail, addUser}= require('./User');
+
+
+module.exports = {
+    checkEmail, 
+    addUser
+};
```

Comment:
`checkEmail` and `addUser` already exported in the index file in `User` folder why you did import them and export them again in the `queries index file`
You can make it shorter by just import them in the` queries index file` with exporting them from each file

Reply:
It's ok @muath-khatib